test_pytest_html.py

```diff
@@ -25,6 +25,23 @@ def run(testdir, path='report.html', *args):
     return result, html
 
 
+def assert_summary_filter(html, test_outcome='Passed', test_outcome_number=1):
+    cb_active = ("<input checked=\"true\""
+                 " data-test-result=\"{0}\""
+                 .format(test_outcome))
+    cb_unactive = ("<input disabled=\"true\"")
```

Comment:
If you had identical checkbox HTML other than the value of disabled, you could do a regex match with a group for the disabled value. This would mean you'd just have one regular expression, but would be able to check the state of disabled.